.github/workflows/MavenCI.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         operating-system: [ubuntu-latest, windows-latest, macos-latest]
-        java-version: ['8', '9', '11', '13']
```

Comment:
9 can be removed as it was non-LTS release and has been deprecated years ago

Reply:
Thanks for making this change. I agree that LTS versions 8, 11, 17 should be the ones that we should run our CI against.
Could you please remove 9 as well along with 13 ?

Reply:
pushed removal of 9 as well

Reply:
one more thing to make builds faster.

not sure what is the purpose of 

these 2 steps 

```
    - name: Clean, build and install
      run: mvn --file pom.xml clean install
    - name: Build package
      run: mvn -B package --file pom.xml
```

mvn install already rans package. [As per maven lifecycle ](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#default-lifecycle) install is after package, so package phase has been already executed when install happened. So I think it is safe to remove that. Will make your builds faster.

Reply:
@mohitpali ok the remove the "package" phase to speed up the builds?